apps/theming-designer/src/components/SemanticSlots.tsx

```diff
@@ -432,33 +58,107 @@ export const SemanticSlots: React.StatelessComponent<ISemanticSlotsProps> = (pro
     );
   };
 
-  const fillVariantSlotsList = (variantType: VariantThemeType, mappingType: {}): JSX.Element[] => {
+  const trimPaletteSlots = (paletteSlots: IPalette): IPaletteSlots => {
+    let trimmedPaletteSlots: IPaletteSlots = {};
+    for (let palette in paletteSlots) {
+      if (
+        palette.startsWith('theme') ||
+        palette.startsWith('neutral') ||
+        palette === 'black' ||
+        palette === 'white' ||
+        palette === 'redDark' ||
+        palette === 'accent'
+      ) {
+        trimmedPaletteSlots[palette] = (paletteSlots as any)[palette];
+      }
+    }
+    return trimmedPaletteSlots;
+  };
+
+  const isASemanticColor = (semanticString: string): boolean => {
+    return (
+      semanticString.startsWith('body') ||
+      semanticString.startsWith('primary') ||
+      semanticString.startsWith('disabled') ||
+      semanticString.startsWith('focus') ||
+      semanticString.startsWith('variant') ||
+      semanticString.startsWith('default') ||
+      semanticString.includes('action') ||
+      semanticString.includes('link') ||
+      semanticString.startsWith('accent') ||
```

Comment:
is accent both a semantic and a palette slot?

Reply:
Yes there are semantic slots that have the string accent in them and accent is a palette slot as well.